lib/xayn_card_view/card_view_listeners_mixin.dart

```diff
@@ -59,36 +57,41 @@ mixin CardViewListenersMixin on CardViewAnimatedState {
 
         pageOffset = pageOffset.clamp(-1, 1);
 
-        final currentDragCounter = _dragStartCounter;
-        final nextIndex = index + pageOffset;
-        var targetPosition =
-            scrollController!.offset - pageOffset * size * fullSize;
+        jump(pageOffset: pageOffset);
+      };
 
-        if ((pageOffset >= 0 && nextIndex <= 1) ||
-            (pageOffset < 0 && nextIndex < 1)) {
-          targetPosition = scrollController!.offset;
-        }
+  Future<void> jump({required int pageOffset}) async {
+    final constraints = lastKnownConstraints;
 
-        if (currentDragCounter == _dragStartCounter) {
-          _updateNow(
-            targetPosition: targetPosition,
-            pageOffset: pageOffset,
-          );
-        }
+    if (constraints == null) return;
```

Comment:
If we don't expect `jump` to be called before having a `lastKnownConstraints`, would it be better to throw an exception (or assert) in that case?

Reply:
done!